modules/voucher/model/voucherinput.class.php

```diff
@@ -223,8 +196,11 @@ function __construct($args) {
             $this->action['voucher_currency_update']      = true;
             unset($this->action['view_linedetails']);
         }
+        if(isset($args['action_postmotpost_save_currency'])) {
```

Comment:
There is no need for this. Use `$args['action_postmotpost_save_currency']` instead